code/modules/vending/autodrobe.dm

```diff
@@ -225,3 +225,39 @@
 /obj/item/vending_refill/autodrobe
 	machine_name = "AutoDrobe"
 	icon_state = "refill_costume"
+
+/obj/machinery/vending/autodrobe/capdrobe
+	name = "\improper Captain's Autodrobe"
```

Comment:
```suggestion
	name = "\improper Condom's dispenser"
```

Reply:
As we all know, the cap is a condom. That is a well established fact. As such it is only natural that it is named in such a way that honors this long established ascertainment.
It is the Dispenser of the condom. The only logical conclusion being naming it the Condom`s dispenser.

Reply:
Comdom*